frontend/app/app/three/page.tsx

```diff
@@ -1,12 +1,12 @@
 'use client';
 
```

Comment:
Am besten gleich mal den "three" ordner in z.B. "demo" umbennenen
Ist ein wenig logischer fürs präsentieren